main/res/values/strings.xml

```diff
@@ -747,23 +747,23 @@
     <string name="init_debug_note">c:geo can generate a lot of debugging information. Activating this setting will affect the stability of c:geo and is only recommended if you are asked to send a log to the developers. Please keep debug logging disabled during normal usage of c:geo!</string>
     <string name="init_debug">Activate debug log</string>
     <string name="init_dbonsdcard_title">Database location</string>
-    <string name="init_dbonsdcard_note">You may store the database of c:geo either system internally or in a directory on your user storage. Having it on the user storage may free some memory, but you may lose a bit of performance and c:geo will not work if the storage is not accessible or you remove the database file.</string>
+    <string name="init_dbonsdcard_note">You may store the database of c:geo either system internally or into a folder on your user storage. We recommend to only activate this option if you need user access to your database file.</string>
     <string name="init_dbonsdcard">On user storage</string>
     <string name="init_dbmove_dbmove">Moving Database</string>
     <string name="init_dbmove_running">Moving Database</string>
     <string name="init_dbmove_success">Successfully moved the database.</string>
     <string name="init_dbmove_failed">Failed to move database</string>
-    <string name="init_datadirmove_datadirmove">Moving Geocache data directory</string>
-    <string name="init_datadirmove_running">Moving Geocache data directory</string>
-    <string name="init_datadirmove_success">Successfully moved Geocache data directory.</string>
-    <string name="init_datadirmove_failed">Moving the Geocache data directory partly or completely failed. Please check old and new directories and move by hand if necessary.</string>
+    <string name="init_datadirmove_datadirmove">Moving Geocache data folder</string>
+    <string name="init_datadirmove_running">Moving Geocache data folder</string>
+    <string name="init_datadirmove_success">Successfully moved Geocache data folder.</string>
+    <string name="init_datadirmove_failed">Moving the Geocache data folder partly or completely failed. Please check old and new folders and move by hand if necessary.</string>
```

Comment:
... "and move missing files by hand" ...
or: "missing data" (to include both files and folders)

Reply:
Done